CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Bug fixes and improvements
 
+- docs: add reference docs for CLI (clsoe #4327) (#4408)
```

Comment:
1. What reference docs are you referring to?

2. Can you add this to the bottom instead of top? (I am guessing there are no other docs entries so far)

Reply:
1. The PR adds a `reference.rst` for CLI docs. It includes docs for CLI configuration file, `config.yaml`, ENV vars and `.env` files. Do I need to make the changelog more elaborate?
2. On it!